exporter/otlpexporter/testdata/config.yaml

```diff
@@ -20,9 +20,8 @@ exporters:
       initial_interval: 10s
       max_interval: 60s
       max_elapsed_time: 10m
-    per_rpc_auth:
-      type: bearer
-      bearer_token: some-token
+#    auth:
```

Comment:
Was this meant to be commented out? Shouldn't the OTLP Exporter tesdata include the configuration for the bearer auth extension, since it was previously configured with bearer authentication?

Reply:
No, it wasn't meant to be commented out - missed out a test - Thanks for the catch